### PR TITLE
Adds AKM-48 and M116.

### PR DESCRIPTION
**Pull request by Andromeda-K22**
The M116 is a fairly nerfed rifle. Might add it to the ballistics weapon crates one day-

The AKM is a 3-round burst with a high fire delay or a semi-auto with the same fire delay. It's intended to be a slightly more gimmicky weapon. Both use specialized magazines.

### CODE REVIEW

**Comment by Andromeda-K22** [comment]
Alllright, why's this conflicting...

**Comment by Andromeda-K22** [comment]
A few more final things to go.

**Comment by Anewbe** [comment]
From what I saw of the initial commit(s) I'm not really encouraged to merge this. New weapons are well and good, but those sprites are an M16 and an AK47, and I'm not really pleased with the idea of adding more modern weapons, over things with different sprites.

Additionally, I'm not really seeing the niche these fill, gameplay wise.

**Comment by Andromeda-K22** [comment]
I'm not going to try and defend these, since i'm running into more issues than anticipated with merging/conflicts. Closing.

**Comment by SpadesNeil** [comment]
BUT THE SPRITES ARE SO NICE.

Well if you don't want them...
giphy

File: code/modules/projectiles/ammunition/magazines.dm [hunk 1]
```diff
@@ -381,6 +381,16 @@
 /obj/item/ammo_magazine/m545saw/empty
 	initial_ammo = 0
 
+/obj/item/ammo_magazine/m545m116 //spades why did you change 5.56 to 5.45 you make no sense
```
Comment (SpadesNeil) [inline code/modules/projectiles/ammunition/magazines.dm]: Because we already have a space AK and the fact that it used 5.56 ammo TRIGGERS ME

**Pull request closed without merging**